### PR TITLE
Add playtest-20230110 announcement

### PR DESCRIPTION
**Pull request by PunkPun**

### CODE REVIEW

File: gamenews [hunk 1]
```diff
@@ -1,4 +1,4 @@
...
+	Content: We are kicking of New Year with a new playtest! We are nearing a full release, and to make sure it is as stable as it can be we highly encourage everybody to play some games and report any issues! A recap of what is new on the playtests:\n\n  • A new and faster pathfinder model\n  • Improved networking code for online games\n  • Measures against spamming and flooding in the chat\n  • The ability to mute players\n  • Community balance updates for all mods\n  • Text notifications for ingame speech announcements\n  • Automatic map reloading in the background\n\nSee the full news post at www.openra.net for more information and to download the new release.
```
Comment (dragunoff) [inline gamenews:4]: typo - should be *"kicking off"*

Comment (dragunoff) [inline gamenews:4]: perhaps the end should read *"to download the new playtest"*

File: news/_posts/2023-01-10-playtest-20230110.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+disqus_id: "playtest-20221223"
+permalink: "/news/playtest-20230110/"
+---
+
+<div style="border-radius: 4px; background-color: #272d2c; padding: 5px">
+<div style="margin: -10px 5px" markdown="1">
+
+We are kicking of New Year with a new playtest!
+
+Notable changes in [Playtest 20230110]({{ '/download' | relative_url }} include:
```
Comment (dragunoff) [inline news/_posts/2023-01-10-playtest-20230110.md:15]: Missing closing bracket:
`[Playtest 20230110]({{ '/download' | relative_url }})`

**Review by dragunoff — CHANGES_REQUESTED** [review]

File: gamenews (hunk 1 above)
Comment (PunkPun) [reply gamenews]: updated

**Review by dragunoff — APPROVED** [review]
LGTM :+1:

File: news/_posts/2023-01-10-playtest-20230110.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+We are kicking off New Year with a new playtest!
+
+Notable changes in [Playtest 20230110]({{ '/download' | relative_url }}) include:
+
+* Fixed sound device detection on linux
+* Fixed Discord game invites
+* Fixed replays locking up when one of the players dropped connection
+* Fixed Dune 2000 service depot being uncapturable
+* Removed crashing tiles from Red Alert temperate tileset bridge template
+* Other minor fixes
```
Comment (abcdefg30) [inline news/_posts/2023-01-10-playtest-20230110.md:22]: We already write "changes include" above, implying that this list isn't complete. So this line can/should be removed. ("Other minor fixes" isn't really a "notable change".)

File: news/_posts/2023-01-10-playtest-20230110.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+For more information see [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog)
```
Comment (abcdefg30) [inline news/_posts/2023-01-10-playtest-20230110.md:24]:
```suggestion
For more information see the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/759df343555c6aa6b52590ead98d1260f51f9572).
```

File: news/_posts/2023-01-10-playtest-20230110.md (hunk 3 above)
Comment (PunkPun) [reply news/_posts/2023-01-10-playtest-20230110.md:22]: I agree

**Review by abcdefg30 — APPROVED** [review]
grafik

grafik

**Pull request merged**